unit_tests/test_tlslite_utils_tripledes.py

```diff
@@ -33,36 +36,38 @@ def test_no_iv(self):
                           b'\x7c\xa1\x10\x45\x4a\x1a\x6e\x57'
                           b'\x7c\xa1\x10\x45\x4a\x1a\x6e\x57'))
 
-    def test_bad_iv(self):
+    def test_too_short_iv(self):
         with self.assertRaises(ValueError):
             Python_TripleDES(
                 bytearray(b'\x7c\xa1\x10\x45\x4a\x1a\x6e\x57'
                           b'\x7c\xa1\x10\x45\x4a\x1a\x6e\x57'
                           b'\x7c\xa1\x10\x45\x4a\x1a\x6e\x57'),
                 b'\x55\xfe\x07\x2a\x73\x51\xa5')
 
-    def test_bad_key_size(self):
+    def test_too_short_key_size(self):
         with self.assertRaises(ValueError):
             Python_TripleDES(b'\x7c\xa1\x10\x45\x4a\x1a\x6e\x57',
                              b'\x55\xfe\x07\x2a\x73\x51\xa5\x00')
+    if PY_VER > (3, ):
+        # DeprecationWarning check should apply only on python3
```

Comment:
we also need to verify that on Python2 no warnings were emitted ([example code](https://stackoverflow.com/a/3892301/462370) that needs to be modified to assert that there were 0 warnings)